src/server/pmix_server_ops.c

```diff
@@ -730,8 +730,10 @@ pmix_status_t pmix_server_fence(pmix_server_caddy_t *cd,
                                        trk->info, trk->ninfo,
                                        data, sz, trk->modexcbfunc, trk);
         if (PMIX_SUCCESS != rc) {
+            pmix_list_remove_item(&trk->local_cbs, &cd->super);
```

Comment:
Do we need this line in `master` as well [here](https://github.com/pmix/pmix/blob/master/src/server/pmix_server_ops.c#L1036-L1039)?

Reply:
There is an alternate solution currently in master, but I think this approach is better - will have to bring it in, remove the current solution, and verify it is still okay.